lldb/bindings/python/python-swigsafecast.swig

```diff
@@ -34,7 +34,7 @@ PythonObject SWIGBridge::ToSWIGWrapper(lldb::BreakpointSP breakpoint_sp) {
 }
 
 PythonObject SWIGBridge::ToSWIGWrapper(const Status& status) {
```

Comment:
Maybe take the object by value and then move it into SBError, so that the caller has a chance to avoid copies (I expect most of them will not want to retain the status object)?